build.gradle

```diff
@@ -76,7 +76,8 @@ ext {
       project(':h2o-genmodel-ext-xgboost'),
       project(':h2o-ext-xgboost'),
       project(':h2o-ext-krbstandalone'),
-      project(':h2o-ext-authsupport'),
```

Comment:
@jakubhava can you please review this PR from SW point of view? are we still going to relocate Jetty in SW?

Reply:
@michalkurka note that module `h2o-ext-authsupport` is not exactly deleted, it is just moved under both jetty modules (if that was your concern)

Reply:
@jakubhava perfect, please make sure the security documentation is consistent